modules/local/benchmarking/location_plots/main.nf

```diff
@@ -0,0 +1,16 @@
+process LOCATION_PLOT {
+    label "process_single"
+
+    conda "bioconda::seaborn=0.11.2"
+    container 'uphl/seaborn'
```

Comment:
Use [Seqera containers](https://seqera.io/containers/) for creating Docker and singularity containers here